### ISSUE TITLE
config for mod blacklist does not work

### ISSUE THREAD

**Issue by aqua-air** [noobanidus/Lootr#227]
loot_modid_blacklist = [] - List of modids whose loot tables shouldn't be converted (in the format of ["modid", "other_modid"])

I have tried 
loot_modid_blacklist = ["apotheosis"] 
and
loot_modid_blacklist = ["modid:apotheosis"] 
yet it does not work. my server continues to load empty loot chests because apotheosis was removed. 
How do i remove loot tables or ignore mods? 
Can you please provide at least one example for writing your configurations? 
![2023-10-10_20 35 10](https://github.com/noobanidus/Lootr/assets/56618632/32fa2aba-b2f7-44ac-a917-8e31d033d7b9)

**Comment by aqua-air** [comment]
Upon further analysis, This issue only happens in chunks that were generated before apotheosis was removed. Newly generated chunks don't seem to have the issue. 

So it's not such a critical issue as i thought before, **but i would still like to suggest a "update / refresh" feature.** 
Perhaps a command that re-selects loot tables for generated chests. This would fix my problem.

**Comment by noobanidus** [comment]
As mentioned on Discord, there really isn't any solution for this.

To clarify, the blacklist for **mod IDs** should just be `["apotheosis", "minecraft", "etc"]`. 

The blacklists only apply to the *conversion* process. If it is already a Lootr chest, it will have no effect. The current behaviour should be the expected behaviour: the mod that has the loot table is removed, so you will get this message.

The notification itself can be disabled in the configuration, if that is what's concerning you.

In reality, without Lootr, you would have the same situation if you had pre-generated (for example) a large world and then removed a mod: any chests that you open that are linked to that mod would just be empty. 

> but i would still like to suggest a "update / refresh" feature.
> Perhaps a command that re-selects loot tables for generated chests.

I'm not really sure how this would work. 

Lootr is unable to tell the difference between a chest that was created with an invalid loot table (perhaps there was a typo in the structure file or in the structure generation code), and one that was removed, so there is no automatic way to determine if it is a "removed loot table" or an "invalid loot table".

Likewise, loot tables are decided by mod authors in structures and structure generation, so there's no way for Lootr to just "guess" at which loot table should replace it.

I did consider a "remapping" system which would allow you to specify "apothesis:loot_table->minecraft:some_other_loot_table", but as the loot table is stored in the block entity itself, and until that chunk is loaded, the information isn't available, so there's no way for Lootr to spit out a list of block entities' locations/loot tables for this to be plausible.

Hopefully that wall of text makes sense!

Basically put: this is just what happens when you remove mods.
